openverse-api/catalog/api/docs/README.md

```diff
@@ -0,0 +1,116 @@
+# Introduction
```

Comment:
Should these docs go to the make handbook, just like the Catalog and Frontend docs?

Reply:
I don't think so, because they kind of _have to_ live here: https://api.openverse.engineering/v1/ and I don't see a strong reason to duplicate the information. We _should_ however add a link and reference to the API docs in the handbook.